.circleci/config.yml

```diff
@@ -0,0 +1,72 @@
+version: 2.1
+
+commands:
+  before_install:
+    steps:
+      - run: |
+          dnf install make -y
+
+  install:
+    steps:
+      - run:
+          name : list_folder
+          command : ls
+      - run:
+          name : install_cmd
+          command: make install
+  
+  before_script:
+    steps:
+      - run: |
+          sudo -u nuodb "/opt/nuodb/etc/nuoadmin" tls $NUO_SET_TLS
+          sudo -u nuodb "/opt/nuodb/etc/nuoadmin" start
```

Comment:
Can you assign a name to this the same way you do with `make install` and other steps that you run? Also, the name does not have to be an identifier. It is a descriptive phrase that appears in the UI, so you can name this "Configure and start nuoadmin" or something. The same applies to the others. You can make all of these descriptive sentences so that it is easy to read the CircleCI display and understand what failed.